meshtastic/tests/test_node_runtime_transport.py

```
@@ -0,0 +1,1233 @@
+
+Tests for admin transport, channel write/delete, ACK/NAK, and position/time command runtimes.
+"""
+
+import logging
+import threading
+import time as time_module
+from typing import Any
+from unittest.mock import MagicMock, patch
+
+import pytest
+from pytest import LogCaptureFixture
```

Ruff (0.15.6)

[warning] Incorrect import of `pytest`; use `import pytest` instead

(PT013)